modules/ROOT/pages/cloudhub-networking-guide.adoc

```diff
@@ -4,30 +4,24 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: cloudhub, cluster, managing, monitoring, runtime manager, arm
 
-image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
-image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
-image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
-image:logo-pcf-disabled.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"]
-
-[NOTE]
-====
-The topics in this document deal only with cloud deployments via the cloud-based version of the Runtime Manager. See xref:deployment-strategies.adoc[Deployment Strategies] for a better understanding of the different possible deployment scenarios.
-====
+This guide is for cloud deployments using the cloud-based version of Runtime Manager. See xref:deployment-strategies.adoc[Deployment Strategies] for information about different deployment scenarios.
```

Comment:
does this need two colons (xref::deployment...) because it's outside the repo? Is it outside the repo?

Reply:
Not outside that repo